test/run_cli.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const ChildProcess = require('child_process');
+const Path = require('path');
+
+const labPath = Path.join(__dirname, '..', 'bin', '_lab');
```

Comment:
That’s [verbatim from the original code](https://github.com/hapijs/lab/blob/b56cac853f9a5fee7e7f0b41d1f9ad89d592ae22/test/cli.js#L29) but I can change it.